src/main/zip/deploy.groovy

```diff
@@ -21,7 +22,8 @@ if(executionGroup != null && !executionGroup.trim().isEmpty()) {
     executionGroups -= [null, ""] // remove empty and null entries
 }
 
-boolean isIncremental = !Boolean.valueOf(props['fullDeploy'])
+String deployType = props['deployType']
```

Comment:
`String deployType = props['deployType']?.trim()`

For safety since you are making it "equal" rather than "contain" `incremental`